upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.12.yaml.template

```
@@ -368,7 +387,7 @@ spec:
           value: {{ . }}
         {{ end }}
 {{ with .Networking.Cilium }}
-        image: "docker.io/cilium/cilium:{{ .Version }}"
+        image: "docker.io/cilium/cilium:{{- or .Version "v.1.7.0" }}"
```

@olemarkus this is missing the typo fix from #8591
```suggestion
        image: "docker.io/cilium/cilium:{{- or .Version "v1.7.0" }}"
```

---

I was thinking both PRs had to be cherry-picked. But that is perhaps a silly. Will fix

---

I agree that both have to be cherrypicked. Just did not see the other one.
Btw, the automated cherry-pick script can do N PRs at same time. Maybe you want to close and create one with both fixes.

---

ah. I didn't know that. That sounds better.